core-bundle/src/EventListener/DataContainer/ContentCompositionListener.php

```diff
@@ -194,6 +194,10 @@ public function renderArticlePasteButton(DataContainer $dc, array $row, string $
 
     private function renderArticlePasteIntoButton(DataContainer $dc, array $row, bool $cr, array $clipboard = null): string
     {
+        if (empty($row['pid'])) {
+            return $this->image->getHtml('pasteinto_.svg').' ';
```

Comment:
@contao/developers Not sure whether we want the "disabled" icon or no icon at all here?

Reply:
🤞🏼 hope that doesn't interfere with https://github.com/terminal42/contao-rootcontent

Reply:
If I’m reading the old code correctly, we rendered the disabled icon: https://github.com/contao/contao/blob/f2651755fce858d40d298ec59dab148bb8af166d/core-bundle/src/Resources/contao/dca/tl_article.php#L781
But we checked for `$row['type'] == 'root'` not for `$row['pid']`